EIPS/eip-4844.md

```diff
@@ -122,7 +122,7 @@ The signature values `y_parity`, `r`, and `s` are calculated by constructing a s
 
 ### Header extension
 
-The current header encoding is extended with a new 256-bit unsigned integer field `excess_data_gas`. This is the running total of excess data gas consumed on chain since this EIP was activated. If the total amount of data gas is below the
+The current header encoding is extended with a new 64-bit unsigned integer field `data_gas_used` and a 256-bit unsigned integer field `excess_data_gas` . This is the running total of excess data gas consumed on chain since this EIP was activated. If the total amount of data gas is below the
```

Comment:
```suggestion
The current header encoding is extended with a new 64-bit unsigned integer field `data_gas_used` and a 256-bit unsigned integer field `excess_data_gas`. This is the running total of excess data gas consumed on chain since this EIP was activated. If the total amount of data gas is below the
```